tools/srchtools/srch_strings.c

```diff
@@ -586,7 +585,6 @@ print_strings (const char *filename, FILE *stream, uint64_t address,
 
       buf[i] = '\0';
       fputs (buf, stdout);
-      free(buf);
```

Comment:
This one seems to be the problem because it can loop back around again.  This free should be moved down outside of the loop to around line 603.  Thanks.